tests/tensor/test_special.py

```diff
@@ -206,6 +208,18 @@ def test_factorial(n):
     )
 
 
+def test_logit():
+    _x = vector("x")
+    actual_fn = function([_x], logit(_x))
+
+    x = random_ranged(0, 1, (5,))
+    actual = actual_fn(x)
+    expected = scipy_logit(x)
+    np.testing.assert_allclose(
+        actual, expected, rtol=1e-7 if config.floatX == "float64" else 1e-5
+    )
```

Comment:
Perhaps a bit more readable?

```suggestion
    x = vector("x")
    actual_fn = function([x], logit(x))

    x_test = np.linspace(0, 1)
    actual = actual_fn(x_test)
    expected = scipy_logit(x_test)
    np.testing.assert_allclose(
        actual, expected, rtol=1e-7 if config.floatX == "float64" else 1e-5
    )
```